documentation/examples/rbac-setup.yml

```diff
@@ -0,0 +1,34 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: prometheus
+rules:
+- apiGroups: [""]
+  resources:
+  - nodes
+  - nodes/proxy
+  - services
+  - endpoints
+  - pods
+  verbs: ["get", "list", "watch"]
+- nonResourceURLs: ["/metrics"]
+  verbs: ["get"]
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: prometheus
+  namespace: default
```

Comment:
Ha, I understand your confusion now. The `Role` is indeed namespaced, where the `ClusterRole` is cluster-wide / not namespaced. 

However the `ClusterRoleBinding` allows you to bind the non-namespaced `ClusterRole` to either cluster wide users or to namespaced service accounts. As you can see in the ClusterRoleBinding spec the metadata doesn't contain a namespace. Only the reference to the service account has a namespace.

I've put the _default_ namespace in the examples to serve as just an example. In most of my own templates I use something like `${NAMESPACE}` for the value which I replace with `envsubst`.  I think we need some explanation round this. Perhaps replace the rbac-setup.yml with a markdown file that explains you need to replace the namespace with your own? But perhaps a page with documentation on how to set up Prometheus on Kubernetes is better off in the documentation than the examples folder.

If you can point me in the right direction I can write this documentation page, but I guess it would be better off in a separate PR.